src/main/java/org/isf/menu/gui/UserBrowsing.java

```diff
@@ -121,250 +86,239 @@ public void userUpdated(AWTEvent e) {
 	private JComboBox pbox;
 	private ArrayList<User> pUser;
 	private String[] pColumns = {
-			MessageBundle.getMessage("angal.menu.userm"), 
-			MessageBundle.getMessage("angal.menu.groupm"), 
-			MessageBundle.getMessage("angal.menu.descm") };
-	private int[] pColumnWidth = {70, 70, 150 };
+			MessageBundle.getMessage("angal.userbrowser.user.txt"),
+			MessageBundle.getMessage("angal.userbrowser.group.txt"),
+			MessageBundle.getMessage("angal.userbrowser.description.txt") };
+	private int[] pColumnWidth = { 70, 70, 150 };
```

Comment:
We were using `angal.*.<key>m` format as reminder to use UPPERCASE translations for tables' columns.
Shall we get rid of them, having only one translation, and apply `.toUpperCase()` in those cases?

Reply:
In my review it appears that most, but not all, table columns were upper cased.  But if that is the project convention then I will use `.toUpperCase()` when filling in column headers.